src/popup/components/Modal.vue

```diff
@@ -111,6 +111,8 @@ export default {
 .modal {
   --screen-padding-x: 24px; // Default spacing
   --screen-bg-color: #{variables.$color-bg-modal};
+  --from-bottom-button-bottom-padding: 25px;
+  --normal-button-padding: 48px;
```

Comment:
I have few problems with these 2 lines:
1. naming convention: these paddings are not a button paddings. They are applied to footer. And in general it's better to name things starting from the more generic word and end with most specific. Eg.: `--footer-padding-bottom`.
2. this approach on having 2 separate CSS variables is a rather bad pattern. It is better to have only one variable set here with its default value, then used in line 202, and changed in the modal variants.
3. 48px is a value that should not be used at any situation regarding the bottom space of the .footer. 48px is a space between the buttons and bottom edge in case don't put the buttons into the footer, so the buttons won't be sticky. So maybe having this CSS variable is totally obsolete?

Example:
```
--footer-padding-bottom: 24px;

.footer {
  padding: 8px var(--screen-padding-x) var(--footer-padding-bottom);
}

&.some-variant {
  --footer-padding-bottom: 666px;
}
```